src/pages/Pool-V3/components/ZapInTab/index.tsx

```
@@ -0,0 +1,314 @@
+              <div className={styles.value}>
+                <span>{numberWithCommas(swapFee, undefined, { maximumFractionDigits: 2 })} %</span>
+              </div>
+            </div>
+            <div className={styles.item}>
+              <div className={styles.itemHasTooltip}>
+                <TooltipHover
+                  isVisible={isVisible}
+                  setIsVisible={setIsVisible}
+                  content={<div>Protocol fee, and it's free now!</div>}
+                  position="right"
+                  children={<span>Zap Fee</span>}
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)